introduction.md

```diff
@@ -3,3 +3,8 @@ Greetings members of the Deadly Vipers dojo.
 My name is Diane L. Smith.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?